package.json

```diff
@@ -58,6 +58,7 @@
     "highlight.js": "^9.13.1",
     "http-proxy": "^1.18.0",
     "img-color-extractor": "^1.0.7",
+    "inline-worker": "^1.1.0",
```

Comment:
Because you're using the prebuilt/pre-browserified version of Recorderjs you don't need this dep.
Also, if you update the deps, you also need to commit the new yarn.lock (generated the next time you run `yarn`).
Also, browser deps should be devDependencies, not regular dependencies.

Reply:
Shifted to devDependencies